docs/en_US/deploy-locally.md

```diff
@@ -0,0 +1,207 @@
+# Deploy ob-operator locally
+
+## Hardware requirements
+
+* CPU: 4 Core available
+* Memory: 10 GB available (A Single-node OceanBase cluster takes 8 ~ 9 GB memory)
+* Disk space: 120 GB available
+
+## Software requirements
+
+You should install following two software before we start. Installation instructions could be found on their sites. You can get minkube by brew install minikube if you develop on macOS.
```

Comment:
how about change this line to "You need to install the following softwares before getting started. Installation instructions can be found on their websites."